src/olympia/devhub/templates/devhub/agreement.html

```diff
@@ -27,5 +27,6 @@
   </div>
   {% else %}
     {{ agreement_form.non_field_errors() }}
+    <p><a href="https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Developer_accounts">{{ _('More information on Developer Accounts') }}</a></p>
```

Comment:
This shouldn't include `en-US` imho

Reply:
Per policy from https://github.com/mozilla/addons-server/issues/5897 it should